docs/getting_started.md

```diff
@@ -4,44 +4,51 @@
+- Learn which factors drive each prediction
 
 ## Installation
 
 ### Windows
 
 - If you haven't, install 64-bit Python 3.5 via [the Anaconda distribution](https://www.continuum.io/downloads)
 - Open the terminal (i.e., CMD or PowerShell, if using Windows)
-- Run `conda install pyodbc`
-- Upgrade to latest scipy (note that upgrade command took forever)
+- **Optional** If you intend to work with MSSQL databases, run `conda install pyodbc`
+- Upgrade to latest scipy
 - Run `conda remove scipy`
 - Run `conda install scipy`
 - Run `conda install scikit-learn`
-   Install healthcareai using **one and only one** of these three methods (ordered from easiest to hardest).
+- Install healthcareai using **one and only one** of these three methods (ordered from easiest to hardest).
      1. **Recommended:** Install the latest release with conda by running `conda install -c catalyst healthcareai`
      2. Install the latest release with pip run `pip install healthcareai`
      3. If you know what you're doing, and instead want the bleeding-edge version direct from our github repo, run `pip install https://github.com/HealthCatalystSLC/healthcareai-py/zipball/master`
 
 #### Why Anaconda?
 
 We recommend using the Anaconda python distribution when working on Windows. There are a number of reasons:
+
 - When running anaconda and installing packages using the `conda` command, you don't need to worry about [dependency hell](https://en.wikipedia.org/wiki/Dependency_hell), particularly because packages aren't compiled on your machine; `conda` installs pre-compiled binaries.
 - A great example of the pain the using `conda` saves you is with the python package **scipy**, which, by [their own admission](http://www.scipy.org/scipylib/building/windows.html) *"is difficult"*
 
 ### Linux
 
 You may need to install the following dependencies:
 - `sudo apt-get install python-tk`
-- `sudo pip install pyodbc`
+- **Optional** If you intend to work with MSSQL databases, run `sudo pip install pyodbc`
     - Note you'll might run into trouble with the `pyodbc` dependency. You may first need to run `sudo apt-get install unixodbc-dev` then retry `sudo pip install pyodbc`. Credit [stackoverflow](http://stackoverflow.com/questions/2960339/unable-to-install-pyodbc-on-linux)
 
-Once you have the dependencies satisfied run `pip install healthcareai` or `sudo pip install healthcareai`
+- Once you have the dependencies satisfied install healthcareai using **one and only one** of these three methods (ordered from easiest to hardest).
+     1. **Recommended:** Install the latest release with conda by running `conda install -c catalyst healthcareai`
+     2. Install the latest release with pip run `pip install healthcareai` or or `sudo pip install healthcareai`
+     3. If you know what you're doing, and instead want the bleeding-edge version direct from our github repo, run `pip install https://github.com/HealthCatalystSLC/healthcareai-py/zipball/master`
```

Comment:
At first I thought this was a function of the randomness in randomized grid search. (Which it was). However, it turns out I left the number of samples in randomized search to **only 2!**, so it was not doing a very good search, and KNN for some reason exposed that.

To fix that, I'm adding a new parameter to the advanced trainer so they can be configured appropriately for the size of hyperparameter grid space and the time/quality tradeoff.

Stellar catch!